terraform/implementation/ecs/README.md

```diff
@@ -31,6 +31,7 @@
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
+| <a name="input_alb_internal"></a> [alb\_internal](#input\_alb\_internal) | Whether the ALB is public or private | `bool` | `true` | no |
```

Comment:
Non-blocking, for your consideration: the concepts of "internal" and "private" may be disjoint to certain end users or jurisdictions. You may want to consider buffing the description to include something like, "Whether the ALB is _public_ (intended for external access) or _private_ (only intended to be accessed within your AWS private cloud)."

Reply:
I'll include these clarifications on the next PR!